docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,726 @@
+>to be able to clone the projects.
+
+The links you'll want to use are:
+- Frontend: `https://github.com/Code-4-Community/jumpstart-frontend.git`
+- Backend: `https://github.com/Code-4-Community/jumpstart-backend.git`
+
+After the projects are open, you'll want to view the starter code by entering the following command into the terminal
+at the bottom of your IntelliJ window for both the frontend and backend projects.
+
+```
+git checkout starter-code
+```
+
+If you don't run the above command, then what you'll probably be working with is a later-stage version of the project.
+That's alright, and you're free to use that later-stage version for help, inspiration, or even if you don't want to
+do the project but are still interested in seeing examples of it. 
+
+Once you've done that, you're ready to install other programs we'll need and build the projects. If you've encountered
+any issues, please don't hesitate to reach out to us so we can help. It's really common to run into errors
+when setting up a new environment.
+
+### Installing required apps
+
+There are a few other programs you'll need to install before we're done here. 
+
+For the frontend and backend, open up the projects and start reading the README.md files by 
+double clicking them in the project pane on the upper left. If you don't see it, you may have to
+expand the pane by clicking on the vertically spelled "project" tab, and then opening up
+the dropdowns indicating folders.
+
+To make sure the formatted text pops up, once you open the README.md files, click on the image-like 
+icon in the upper right to view the formatted markdown. You can also try viewing it side-by-side as
+plaintext and formatted markdown if you're interested to see what it looks like.
+
+Troubleshooting: 
+
+- For the frontend, don't forget to install your dependencies before starting the project, or you'll
+get errors. 
+- For the backend, make sure that you set the `JAVA_HOME` variable and that everything else required
+is installed.
+    - `JAVA_HOME` is a system-level variable which lets programs know where to look for the JDK (Java Development Kit). 
+- For Git bash, the terminal may need to be manually set in every project. Don't forget to do that, or
+else you'll end up running commands that don't exist in Windows Command Prompt.
+    - Git bash is a program which lets you run a Mac/Linux like terminal on Windows. We describe it  little more below.
+
+## Terminal Basics
+
+The terminal is a really important tool, which you'll find extremely useful throughout your 
+developer career. Not only does it allow you to access settings and commands which you would otherwise
+not have access to, it sometimes is just easier to be able to do some things by typing out commands
+rather than pointing and clicking with a mouse. 
+
+Learning how to use the terminal well may take a while, but we'll get you started with a few commands
```

Comment:
Get rid of the "well" in the sentence, sounds awkward

Reply:
Fine :angery: